src/main/java/cn/nukkit/network/protocol/NPCRequestPacket.java

```diff
@@ -5,18 +5,54 @@
 @ToString
 public class NPCRequestPacket extends DataPacket {
 
+    public long entityRuntimeId;
+
+    public int requestType;
+    
+    public String commandString;
+
+    public int actionType;
+    
+    public enum Request {
+    
+        SET_ACTIONS(0),
+        EXECUTE_ACTION(1),
+        EXECUTE_CLOSING_COMMANDS(2),
+        SET_NAME(3),
+        SET_SKIN(4),
+        SET_INTERACTION_TEXT(5);
+    
+        private final int value;
```

Comment:
There's no point in storing the value since it's the same as the constant ordinal